packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -172,6 +177,96 @@ export const transformPlotlyJsonToGVBCProps = (
   };
 };
 
+// TODO: Add support for padding in continuous axis
+// FIXME: Vertical bar chart displays duplicate legend buttons
```

Comment:
lets move these to the document

Reply:
Added these tasks in the document. Should I remove them from the code?

Reply:
yes

Reply:
done